### PR TITLE
Start assignment

### PR DESCRIPTION
**Pull request by ShaunWMusic**

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
- [x] Create a new project with `yo qunit-broccoli` called `11 Spotify Collection`
- [x] Copy the JSON data and SASS from this gist: https://gist.github.com/rtablada/dc5a928e8dec61830fc5333726d871b4
- [x] Use npm to install the `bulma` package (don't forget to add `--save`)
- [x] Commit these changes to the `master` branch and push to github
- [x] Change to the `develop` branch

**Comment by ShaunWMusic** [comment]
- [x] Import the data from the json file into `index.js`
- [x] Create a new model module for `spotify-track` that exports a `class`
  - [x] Make a constructor that takes in a `data` argument
    - [x] Set properties on the model instance for:
      - [x] `artist`
      - [x] `name`
      - [x] `image`
      - [x] `url` this should a url like "https://open.spotify.com..."
- [x] Create a new view module for `spotify-track` that exports a `class`
  - [x] Make a constructor that takes in a `model` argument
    - [x] Create the wrapping element to represent a single `track`
    - [x] Add classes for `panel-block` and `track` to the wrapping element
    - [x] Create the skeleton HTML within the view element
    - [x] Set the `model` property on the instance based on the `model` argument
  - [x] Make a render method on the class
    - [x] Fill an element with a class of `track__name` with the `name` property from the model stored on the current view instance
    - [x] Fill an element with a class of `track__artist` with the `artist` property from the model stored on the current view instance
    - [x] Set the `src` an element with a class of `track__img` with the `image` property from the model stored on the current view instance
    - [x] Set the `href` on an element with a class of `track__link` with the `name` property from the model stored on the current view instance
- [x] To the `index.html` file add the following elements:
  - [x] Div with classes: "section" and "page"
    - [x] Div with classes: "panel" and "is-fullwidth"
      - [x] Paragraph with a class "panel-heading"
      - [x] Div with a class "results"
- [x] In `index.js`:
  - [x] import the view and model classes
  - [x] Select the `results` element and set it to a variable
  - [x] Loop through the track data from the JSON information
    - [x] Within the loop function:
      - [x] Create a `new` instance of the model based on data from the current item in the array
      - [x] Create a `new` instance of a view sending the new model as an argument
      - [x] Add the element from the view to the end of the `results` element
      - [x] Run the `render` method on the view instance

**Comment by ShaunWMusic** [comment]
classes linked 
- [x] page
- [x] panel
- [x] panel-block
- [x] results
- [x] track
- [ ] track__media
- [ ] track__frame
- [ ] track__artist
- [ ] track__name
- [ ] track__img
- [ ] track__link

File: app/spotify-tracker/view.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    this.track = document.createElement('div');
+    this.track.classList.add('panel-block');
+    this.track.classList.add('track');
+    this.track.innerHTML = `
+      <img src="" class="track__img" alt="">
+      <div class"panel-block__content">
+      <h2 class="track__artist"></h2>
+      <h4 class="track__name"></h4>
+      </div>
+      <div class="spotify">
```
Comment (rtablada) [inline app/spotify-tracker/view.js]: This should be something like `panel-block__right` and inside of the `track__link` you will want to have a `span.fa.fa-spotify.fa-2x` to get the spotify link showing up.

**Review by rtablada — CHANGES_REQUESTED** [review]

**Pull request merged**